src/scores/continuous/standard_impl.py

```diff
@@ -100,7 +104,7 @@ def rmse(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
     return _rmse
 
 
-def mae(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
+def mae(fcst: FlexibleArrayType, obs: FlexibleArrayType, reduce_dims: FlexibleDimensionTypes=None, preserve_dims: FlexibleDimensionTypes=None, weights=None):
```

Comment:
I can't put a note on the specific line but the docstrings here need fixing up with these type changes. Now that we are implementing type hints properly the parameter types from the docstring can be removed.